src/common/k8s-api/endpoints/horizontal-pod-autoscaler.api.ts

```diff
@@ -72,6 +72,27 @@ export type HorizontalPodAutoscalerMetricSpec =
   | OptionVarient<HpaMetricType.Pods, BaseHorizontalPodAutoscalerMetricSpec, "pods">
   | OptionVarient<HpaMetricType.ContainerResource, BaseHorizontalPodAutoscalerMetricSpec, "containerResource">;
 
+interface HorizontalPodAutoscalerBehavior {
+  scaleUp?: HPAScalingRules;
+  scaleDown?: HPAScalingRules;
+}
+
+interface HPAScalingRules {
+	stabilizationWindowSecond?: number;
+	selectPolicy?: ScalingPolicySelect;
+	policies?: HPAScalingPolicy[];
+}
+
+type ScalingPolicySelect = string;
+
+interface HPAScalingPolicy {
+	type: HPAScalingPolicyType;
+	value: number;
+	periodSeconds: number;
+}
```

Comment:
_nit:_ too big indentation within both `interface`s (comparing to one above)

Reply:
There were tabs instead of spaces. Surprisingly, linter ignores that. Fixed.